test/built-ins/Symbol/asyncIterator/name-prop.js

```diff
@@ -0,0 +1,11 @@
+/*---
+author: Benjamin Gruenbaum <benjamingr@gmail.com>
+esid: early
+description: Name prop of Symbol.asyncIterator
+info: >
+  The value of the name property of this function is "[Symbol.asyncIterator]".
+features: [Symbol.asyncIterator]
+---*/
+
+
+assert.sameValue(Symbol.asyncIterator.name, '[Symbol.asyncIterator]');
```

Comment:
"[Symbol.asyncIterator]" is not a name of the symbol, but the method of %AsyncIteratorPrototype%
https://tc39.github.io/proposal-async-iteration/#sec-asynciteratorprototype-asynciterator